src/components/partnership-producing-medium/partnership-producing-medium.service.ts

```diff
@@ -30,13 +35,13 @@ export class PartnershipProducingMediumService {
     engagement: LanguageEngagement,
     session: Session,
   ): Promise<SecuredPartnershipsProducingMediums> {
-    const perms = await this.auth.getPermissions({
-      resource: IProject,
-      sessionOrUserId: session,
-      sensitivity: engagement.sensitivity,
-      otherRoles: engagement.scope,
-    });
-    if (!perms.partnership.canRead) {
+    const adjustedContext = withEffectiveSensitivity(
+      withScope(engagement, ['member:true']),
+      Sensitivity.High,
+    ) as any;
```

Comment:
Sorry this was just an example I was giving in the previous comment.
We don't want to hardcode these context values.
```suggestion
    const adjustedContext = engagement as any;
```